File: practice/16/python/16.py
n = int(input("Введите количество билетов:\n"))
found = 0
for numcount in range (n):
    tick = str(input("Введите номер билета/билетов:\n"))
    if (tick[0] == 'a') and (tick[4] == '5') and (tick[5] == '5') and (tick[6] == '6') and (tick[7] == '6') and (tick[8] == '1'):
        print(tick, end = ' ')
        break
    else:
        found += 1
if found == n:
    print("-1", end = ' ')

File: practice/16/c++/16/16/16.cpp
#include <iostream>
#include <string>
using namespace std;

int main() {
    setlocale(LC_ALL, "Russian");
    int n, found = 0;
    string tick;
    cout << "Введите количество билетов:\n"; cin >> n;
    cout << "Введите номер билета/билетов:\n";
    for (int numcount = 1; numcount <= n; numcount++) {
        cin >> tick;
        if (tick[0] == 'a' && tick[4] == '5' && tick[5] == '5' && tick[6] == '6' && tick[7] == '6' && tick[8] == '1') {
            cout << tick << ' ';
        }
        else {
            found += 1;
        }
    }
    if (found == n) {
        cout << "-1";
    }
}